src/main/java/com/rocana/lucene/codec/v1/RocanaPerFieldPostingsFormat.java

```diff
@@ -274,6 +304,30 @@ public FieldsReader(final SegmentReadState readState) throws IOException {
       this.segment = readState.segmentInfo.name;
     }
 
+    /**
+     * Return our fork of Lucene's postings format.
+     *
+     * The original Lucene code did an SPI (Service Provider Interface) lookup to
+     * get the postings format. The problem is we forked the postings format, but
+     * didn't register it via SPI -- for the reason see:
+     * {@link RocanaSearchCodecV1#getActualPostingsFormat()}
+     *
+     * This meant our custom codec was using our forked postings format, but here
+     * the SPI lookup returned Lucene's original postings format. So we're mixing
+     * postings formats.
+     *
+     * In this method we'll catch those lookup calls for Lucene's postings format
+     * ("Lucene50") and instead return Rocana's fork.
+     */
+    private PostingsFormat lookupPostingsFormat(String postingsFormatShortName) {
+      if (postingsFormatShortName.equals("Lucene50")) {
+        RocanaSearchCodecV1 codec = (RocanaSearchCodecV1) Codec.forName(RocanaSearchCodecV1.NAME);
```

Comment:
@mpeterson0x0 I wanted to get "the one and only instance of our postings format" which we could do by looking it up in SPI, but since we don't have store our postings format in SPI (yet) it seemed like a good alternative to get the one and only instance of our codec and then use that to get the `final` postings format field.